modules/SemanticWebImport/src/main/java/fr/inria/edelweiss/semantic/analyzer/HTTPRequestPoster.java

```diff
@@ -0,0 +1,153 @@
+
+package fr.inria.edelweiss.semantic.analyzer;
+
+/**
+ *
+ * @author ukhy
+ */
+
+
+import java.io.*;
+import java.net.HttpURLConnection;
+import java.net.ProtocolException;
+import java.net.URL;
+import java.net.URLConnection;
+
+
+public class HTTPRequestPoster {
+    
+    private String result;
+    private String URI;
+    
+    public HTTPRequestPoster(String URI){
+        this.URI = URI;
+    }
+    
+    public String getResult(){
+        return this.result;
+    }
+    public void setURI(String stringURI){
+        this.URI = stringURI;
+    }
+    
+    public String getURI(){
+        return this.URI;
+    }
+    
+    public static String sendGetRequest(String endpoint, String requestParameters) {
+        String result = null;
+        if (endpoint.startsWith("http://")) {
+        // Send a GET request to the servlet
+            try
+            {
+                // Construct data
+            StringBuffer data = new StringBuffer();
+                // Send data
+            String urlStr = endpoint;
+            if (requestParameters != null && requestParameters.length () > 0)
+                {
+                urlStr += "?" + requestParameters;
+                }
+            final URL url = new URL(urlStr);
+            final URLConnection conn = url.openConnection ();
+            conn.addRequestProperty("Accept", "application/rdf+xml");
+            // Get the response
+            final InputStream inputStream = conn.getInputStream();
+            final BufferedReader rd = new BufferedReader(new InputStreamReader(inputStream, "UTF-8"));
+            StringBuffer sb = new StringBuffer();
+            String line;
+            while ((line = rd.readLine()) != null)
+                {
+                sb.append(line);
+                }
+            rd.close();
+            result = sb.toString();
+            } catch (Exception e)
+                {
+                e.printStackTrace();

```

Comment:
All calls to e.printStackTrace should be changed to
```
Logger.getLogger(JSONExporter.class.getName()).log(Level.SEVERE, null, e);
```

or 

```
Exceptions.printStackTrace(e);
```
like in other parts of your code.